go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/aws/amazon-cloudwatch-agent
 
-go 1.18
+go 1.19
```

Comment:
Did you run go mod tidy after changing the version?

Reply:
It did not change anything for me.